tests/scene/test_gltf_document.h

```diff
@@ -0,0 +1,256 @@
+/**************************************************************************/
+/*  test_gltf_document.h                                                  */
+/**************************************************************************/
+/*                         This file is part of:                          */
+/*                             GODOT ENGINE                               */
+/*                        https://godotengine.org                         */
+/**************************************************************************/
+/* Copyright (c) 2014-present Godot Engine contributors (see AUTHORS.md). */
+/* Copyright (c) 2007-2014 Juan Linietsky, Ariel Manzur.                  */
+/*                                                                        */
+/* Permission is hereby granted, free of charge, to any person obtaining  */
+/* a copy of this software and associated documentation files (the        */
+/* "Software"), to deal in the Software without restriction, including    */
+/* without limitation the rights to use, copy, modify, merge, publish,    */
+/* distribute, sublicense, and/or sell copies of the Software, and to     */
+/* permit persons to whom the Software is furnished to do so, subject to  */
+/* the following conditions:                                              */
+/*                                                                        */
+/* The above copyright notice and this permission notice shall be         */
+/* included in all copies or substantial portions of the Software.        */
+/*                                                                        */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,        */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF     */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. */
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY   */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,   */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
+/**************************************************************************/
+
+#ifndef TEST_GLTF_DOCUMENT_H
+#define TEST_GLTF_DOCUMENT_H
+
+#include "modules/gltf/extensions/gltf_document_extension_convert_importer_mesh.h"
+#include "modules/gltf/gltf_document.h"
+#include "scene/main/node.h"
+
+#include "tests/test_macros.h"
+#include "tests/test_utils.h"
+
+namespace TestGLTFDocument {
+
+struct GLTFArraySize {
+	String key;
+	int val;
+};
+
+struct GLTFKeyValue {
+	String key;
+	Variant val;
+};
+
+struct GLTFTestCase {
+	String filename;
+	String copywrite;
+	String generator;
+	String version;
+	Vector<GLTFArraySize> array_sizes;
+	Vector<GLTFArraySize> json_array_sizes;
+	Vector<GLTFKeyValue> keyvalues;
+};
+
+const GLTFTestCase glTF_test_cases[] = {
+	// Test 1
```

Comment:
```suggestion
```

I recommend not numbering the tests. In a previous place, I had numbered text up to 16 counts, and I had to remove them because it was hard to maintain.